smdebug/core/collection.py

```diff
@@ -45,6 +45,10 @@ class CollectionKeys:
     FULL_SHAP = "full_shap"
     TREES = "trees"
 
+    @classmethod
+    def values(cls):
+        return [value for name, value in vars(cls).items() if name.isupper()]
```

Comment:
Risky to do it this way. Predictions for example is a valid collection user might want to create for TF/PT/MX. But since it's in the 1p collections for xgboost, your check won't be applied for that.